react-search-component/my-app/src/FilteredItems.js

```diff
@@ -0,0 +1,24 @@
+export default function FilteredItems(props) {
+  //Lowercase each string
+  const lowerCaseList = props.list.map((element) => element.toLowerCase());
+  const lowerCaseInputString = props.string.toLowerCase();
+
+  //Create array of filtered <li> element with each string
+  const filteredList = [];
+
+  console.log(props.string);
+
+  for (let i = 0; i < props.list.length; i++) {
```

Comment:
I suggest you use `props.list.map`. Standard for-loops are rarely used by experienced JS programmers.